src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalThrowsCheck.java

```diff
@@ -60,6 +60,15 @@
  * <pre>
  * &lt;module name="IllegalThrows"/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class Test {
+ *  public void func1() throws RuntimeException {} // violation
+ *  public void func2() throws Exception {}  // ok
+ *  public void func3() throws Error {}  // violation
+ *  public void func4() throws Throwable {} // violation
```

Comment:
This should be indented with 2 spaces to match other examples.

Reply:
Done